test/valid/valid.isTritArray.js

```diff
@@ -0,0 +1,32 @@
+var chai = require('chai')
+var assert = chai.assert
+var inputValidator = require('../../lib/utils/inputValidator.js')
+var isTritArray = inputValidator.isTritArray
+
+describe('utils.isTransactionHash', function () {
+  it('should return true for input of Array type', function () {
+    assert.strictEqual(isTritArray([-1, 0, 1]), true)
+  })
+
+  it('should return true for input of Int8Array type', function () {
+    assert.strictEqual(isTritArray(new Int8Array(1).fill(1)), true)
+  })
+
+  it('should return false for input of invalid trits', function () {
+    assert.strictEqual(isTritArray([-1, 0, 2]), false)
+    assert.strictEqual(isTritArray(new Int8Array(1).fill(2)), false)
+  })
+
+  it('should return true for empty array', function () {
```

Comment:
Should it return true or maybe length should also be checked?

Reply:
In case user doesn't specify a `length`,  trit array can be of any length, even empty.

Reply:
It should probably be:
it('should return true for empty array, if length is not specified...